### DISCUSSION TITLE
v3.2.32

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#11377] [General]
## What's Changed
* feature: Collapsed sidebar group dropdowns by @danharrin in https://github.com/filamentphp/filament/pull/11376
* Add a contributing page to the docs by @dissto in https://github.com/filamentphp/filament/pull/11348
* Add Turkish Translations for Import/Export Action by @remzikocak in https://github.com/filamentphp/filament/pull/11368
* Add Polish export/import translations by @shepard153 in https://github.com/filamentphp/filament/pull/11366

## New Contributors
* @shepard153 made their first contribution in https://github.com/filamentphp/filament/pull/11366

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.31...v3.2.32

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.32'>v3.2.32</a>.</em>

**Comment by CyberPunkCodes** [comment]
This breaks installation. Please see the following:

Issue (closed by bot due to silly no-repo rules): https://github.com/filamentphp/filament/issues/11382

Proposed PR: https://github.com/filamentphp/support/pull/3

Commit where the sky fell (SupportServiceProvider.php): https://github.com/filamentphp/support/commit/293e18a96ee4ccfe19ec25221ed9580937bb482e#diff-d6fb58c25d2d95e5fdc3f0da7940ccc5d3dd7ffb9e87553f6ddc5891b6832693
